Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -69,9 +78,138 @@
     }
   }
 
+  .navigation {
+    display: flex;
+
+    .nav-list {
+      padding-left: 10%;
+      width: 80%;
+    }
+
+    .cart-icon {
+      width: 20%;
+      padding-left: 0;
+    }
+
+    .site-nav {
+      display: flex;
+      justify-content: space-evenly;
+      list-style: none;
+
+      a {
+        color: $color-categories-text;
+        text-decoration: none;
+
+        &:hover {
+          color: $color-text-hover;
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1024px) {
+  .sidenav {
+    display: none;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .header-mobile-section {
+    display: none;
+  }
+
+  .section-header {
+    padding: 15px;
+
+    header {
+      display: flex;
+      margin-top: 21px;
+      margin-bottom: 21px;
+
+      .header-logo {
+        display: flex;
+        justify-content: center;
+        width: 50%;
+        margin-top: 0;
```

Comment:
If you have no margins on left and right you could use `margin: 0;` or if you DO have margins on left and right you could use `margin: 0 Xpx;` where X is the size of the margin on both left and right.